dev/fattest.simplicity/src/componenttest/containers/ExternalDockerClientFilter.java

```diff
@@ -70,7 +71,10 @@ public boolean isForced() {
     @Override
     public boolean isMatched(ExternalTestService dockerService) {
         String m = "isMatched";
+        this.hostname = dockerService.getHostname();
```

Comment:
I would wait to set this until after we have determined if we are going to filter out this host or not. 

Move this to line 95 after certificates are validated.